charts/seed-bootstrap/dashboards/fluent-bit-dashboard.json

```
@@ -731,7 +742,98 @@
       ],
       "timeFrom": null,
       "timeShift": null,
-      "title": "[Output Plugin] Incoming logs",
+      "title": "[Output Plugin] Incoming Logs",
+      "transformations": [
+        {
+          "id": "reduce",
+          "options": {
+            "reducers": [
+              "delta"
+            ]
+          }
+        },
+        {
+          "id": "organize",
+          "options": {
+            "excludeByName": {},
+            "indexByName": {},
+            "renameByName": {
+              "Delta": "Total number of logs",
+              "Field": "Namespace",
+              "Last": "Dropped logs from last 1h",
+              "Mean": "Average number of incoming logs per hour",
+              "Total": ""
+            }
+          }
+        },
+        {
+          "id": "groupBy",
+          "options": {
+            "fields": {
+              "Namespace": {
+                "aggregations": [],
+                "operation": "aggregate"
+              }
+            }
+          }
+        }
+      ],
+      "type": "table"
+    },
+    {
+      "datasource": null,
+      "description": "Table which shows the total incoming logs with attached endpoint  and are ready to be forwarded to the Promtail client",
```

```suggestion
      "description": "Table which shows the total incoming logs with attached endpoint and are ready to be forwarded to the Promtail client",
```
Minor: Duplicate space